lib/iris/analysis/maths.py

```diff
@@ -121,34 +121,24 @@ def _assert_is_cube(cube):
 def _assert_compatible(cube, other):
     """
     Checks to see if cube.data and another array can be broadcast to
-    the same shape using ``numpy.broadcast_arrays``.
+    the same shape using ``numpy.broadcast_arrays`
```

Comment:
This no longer uses numpy.broadcast_shapes.  It uses dask.array.core.broadcast_shapes.  Make this docstring correct.